src/main/java/twilightforest/item/ItemTFMazeMap.java

```diff
@@ -294,7 +296,7 @@ public EnumRarity getRarity(ItemStack par1ItemStack) {
 
 	@Override
 	public String getItemStackDisplayName(ItemStack stack) {
-		return ("" + net.minecraft.util.text.translation.I18n.translateToLocal(this.getUnlocalizedNameInefficiently(stack) + ".name").trim() + " #" + stack.getItemDamage());
+		return ("" + TwilightForestMod.proxy.translate(this.getUnlocalizedNameInefficiently(stack) + ".name").trim() + " #" + stack.getItemDamage());
 	}
```

Comment:
actually, looking at vanilla, it no longer appends the map number to the stack's display name, this is legacy copy stuff
You can just remove the `{ItemTFMazeMap, ItemTFMagicMap}.getItemStackDisplayName` methods entirely and the proxy method will become unneeded.